src/features/user/MFV2/MyContributions/index.tsx

```diff
@@ -52,9 +52,9 @@ const MyContributions = ({ profilePageType, userProfile }: ProfileV2Props) => {
       <ListHeader
         profilePageType={profilePageType}
         displayName={
-          userProfile.type === 'individual'
-            ? userProfile.displayName.split(' ').slice(0, 1)[0]
-            : userProfile.displayName
+          userProfile?.type === 'individual'
+            ? userProfile?.displayName.split(' ').slice(0, 1)[0]
+            : userProfile?.displayName
```

Comment:
Is the null check on `userProfile` needed? Can it ever be `null` or `undefined`?

Reply:
I saw the error stating `cannot read properties of undefined` while I was working in the PR though it does not seem to occur now but I'm unsure about the steps to reproduce the same. Though, it does not seem to occur now so I've removed the check for now

Reply:
Should be safe as we're checking for profile before rendering `MyContributions` in any case.